WordPressVIPMinimum/Sniffs/VIP/RestrictedFunctionsSniff.php

```diff
@@ -244,4 +255,49 @@ public function getGroups() {
 
 		return $groups;
 	}
+
+	/**
+	 * Verify is the current token is a function call.
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return bool
+	 */
+	public function is_targetted_token( $stackPtr ) {
```

Comment:
This method doesn't seem to be in use?

Reply:
@GaryJones See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/00f5d8989f4c6b6c592c54e202aa7ef608988b88/WordPress/AbstractFunctionRestrictionsSniff.php#L213

Reply:
1. Does this method differ from the method in the parent class?

  If not, adding it here as well adds nothing.

  If it does differ, then it should be clearly identified how it differs.

2. Where is this method called in this class?

Reply:
> 1. Does this method differ from the method in the parent class?

Yes -- it differs by checking if the restricted function in question is part of a specific class before flagging it.

> 2. Where is this method called in this class?

Right here in the parent class: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/00f5d8989f4c6b6c592c54e202aa7ef608988b88/WordPress/AbstractFunctionRestrictionsSniff.php#L199

Reply:
> 1. Does this method differ from the method in the parent class?

Yes -- it differs by checking if the restricted function in question is part of a specific class before flagging it.

> 2. Where is this method called in this class?

Right here in the parent class: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/00f5d8989f4c6b6c592c54e202aa7ef608988b88/WordPress/AbstractFunctionRestrictionsSniff.php#L199

Reply:
Ah,got it, thanks.